site/docs/foundations/duration.mdx

```diff
@@ -20,21 +18,78 @@ There are four durations available to choose from: Instant, Perceptible, Notable
 | `--salt-duration-notable`     | 1000                    |
 | `--salt-duration-cutoff`      | 10000                   |
 
-### Instant
+## Instant (0-100ms)
```

Comment:
If we do change this to 0ms, do we still want to say 100ms in this section? @pseys

Reply:
I can also add the update into this PR if we confirm changing it to 0

Reply:
I made the change in this PR, cc @navkaur76 @pseys